website/templates/profile/oauth_app_detail.mako

```diff
@@ -0,0 +1,117 @@
+<%inherit file="base.mako"/>
+<%def name="title()">Application Detail</%def>
+<%def name="content()">
+<style type="text/css">
+    .border-box {
+        padding: 15px 10px;
+        border: solid #DDD;
+        border-width: 1px 0;
+        line-height: 1.1;
+        display: block;
+        margin-bottom: 1em;
+    }
+</style>
+
+<h2 class="page-header">Application Detail</h2>
+
+
+<div id="applicationDetailPage" class="row">
+    <div class="col-sm-3 affix-parent">
+      <%include file="include/profile/settings_navpanel.mako"/>
+    </div>
+
+    <div class="col-sm-9 col-md-7">
+        <div class="row">
+            <div class="col-sm-12">
+                <div class="breadcrumb"><i class="fa fa-angle-double-left"></i> <a href="${web_url_for('oauth_application_list')}">Return to list of registered applications</a></div>
```

Comment:
Isn't the application list URL an attribute of the ViewModel? If so, then use Knockout to bind it to the `a` element. When possible, try not to mix KO and mako.